testsuite/tests/cpp-api/cpp.ml

```diff
@@ -0,0 +1,15 @@
+(* TEST
+   modules = "cpp_stubs1.c cpp_stubs2.c cpp_stubs3.c";
+   readonly_files = "all-includes.h";
+   flags = "-ccopt -x -ccopt c++ -ccopt -std=c++11";
```

Comment:
These flags work with GCC and Clang, but probably not with MSVC.  Can the test be turned off for MSVC?

Reply:
Yes - leftover from #13476, we have a `not-msvc` action!